recipes/Functions.py

```
@@ -517,25 +555,26 @@ def extract_gcs_urls(
     code: str,
     variables: dict[str, typing.Any],
     env: dict[str, str] | None,
     gooey_memory: dict[str, typing.Any] | None,
+    package_json: typing.Any | None,
     response: "FunctionsPage.ResponseModel",
 ) -> dict[str, typing.Any] | None:
-    tag = f"run_id={sr.run_id}&uid={sr.uid}"
-
-    # this will run functions/executor.js in deno deploy
+    # this will run functions/executor_cf in a cloudflare dynamic worker
     r = requests.post(
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)